packages/client/hmi-client/src/components/sidebar-panel/simulation-result-sidebar-panel.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+import { Project } from '@/types/Project';
+import { useRouter } from 'vue-router';
+
+defineProps<{
+	project: Project;
```

Comment:
Why not just send `simulation_runs` as prop?

Reply:
It is a balance between least knowledge principle / demeter's law, and to have a unified interface. 

In this case because all the sidebar panels are project-assets:
- it looks more unified to pass in project object to everything rather than their specific asset-classes
- we will probably need to add/remove project-associations, in which case we will want to have a project-id handy

Reply:
Adding/removing assets within a project should be done via the `services/projects.ts` and not within the component.
I'm just worried of having to much reactivity on massive object when it is not needed for now.
I prefer to keep it simple until we need more info.

Reply:
Agreed, but I need to do something like
```
projectService.removeSimulation(projectId, simulationPlanId)
```

So the components needs `projectId` and `simulation id array` passed in as properties. So we might as well pass in the project data structure that encapsulates both.